### DISCUSSION TITLE
Remain Logged Into Documentation Site

### DISCUSSION THREAD

**Discussion by benjibee** [livewire/flux#188] [Feature Requests] [answered]
Lots of code examples cannot be viewed without being logged in, but my sessions don't seem to be persisting. So now I find myself having to login each time I visit the docs and it really breaks up the workflow.

It would be great to have a "Remember Me" button when logging in that keeps me logged in for a reasonable amount of time.

**Comment by calebporzio** [comment, marked as answer]
I just bumped up the session lifetime to 2 days instead of 2 hours.

**Reply by benjibee** [reply]
Thanks, Caleb!

Would you consider something a bit more generous like 2-weeks? Sticking with your Lego bin analogy I'll be referencing the docs very often while coding and seeing that lock icon when you've already logged in not that long ago just doesn't seem quite right.

**Reply by benjibee** [reply]
@joshhanley I was waiting on an answer here before marking this as answered.

**Reply by joshhanley** [reply]
@benjibee Ah yep, sorry just saw Caleb had changed it. Have reopened

**Reply by benjibee** [reply]
@joshhanley It's been more than two days now and I'm still logged in, I guess we can consider this fixed :)
